boards/arm/b_l072z_lrwan1/doc/b_l072z_lrwan1.rst

```diff
@@ -0,0 +1,225 @@
+.. _b_l072z_lrwan1_board:
+
+ST B-L072Z-LRWAN1 Discovery kit
+###############################
+
+Overview
+********
+
+This Discovery kit features an all-in-one open module CMWX1ZZABZ-091 (by Murata).
+The module is powered by an STM32L072CZ and an SX1276 transceiver.
+
+This kit provides:
+
+- CMWX1ZZABZ-091 LoRa |reg| /Sigfox |trade| module (Murata)
```

Comment:
We generally recognize third-party trademarks with a simple asterisk instead of a TM or (R), so you can change these references to simply ``LoRa*``, etc.